components/scream/src/share/atm_process/atmosphere_process.hpp

```diff
@@ -332,16 +389,23 @@ class AtmosphereProcess : public ekat::enable_shared_from_this<AtmosphereProcess
     f.get_header_ptr()->get_tracking().add_customer(weak_from_this());
   }
 
-  virtual void set_required_field_impl (const Field<const Real>& f) = 0;
-  virtual void set_computed_field_impl (const Field<      Real>& f) = 0;
+  virtual void set_required_field_impl (const Field<const Real>& f) {};
```

Comment:
We should add a comment on top of these two methods, explaining that subclasses _can_ override this if they need to do additional stuff (like additional book keeping or setup), but it is not necessary.